OpenSim/Actuators/CoordinateActuator.h

```diff
@@ -57,6 +57,11 @@ OpenSim_DECLARE_CONCRETE_OBJECT(CoordinateActuator, ScalarActuator);
         "Name of the generalized coordinate to which the actuator applies.");
     OpenSim_DECLARE_PROPERTY(optimal_force, double,
         "The maximum generalized force produced by this actuator.");
+//==============================================================================
+// OUTPUTS
+//==============================================================================
+    OpenSim_DECLARE_OUTPUT(force, double, computeActuation,
+                           SimTK::Stage::Acceleration);
```

Comment:
What about naming the output `generalized_force` or `actuation`? The name `force` could be confusing, since the CoordinateActuator may actually be applying a moment (this is actually more common). I know that this is already a problem with the `optimal_force` property, but one may expect output names to be more precise.